WooCommerce/Classes/ViewRelated/Top Banner/ErrorTopBanner.swift

```diff
@@ -0,0 +1,83 @@
+import SwiftUI
+
+/// SwiftUI view equivalent to the view created by `ErrorTopBannerFactory`.
+/// TODO: if needed - convert this view to a reusable top banner.
+struct ErrorTopBanner: View {
```

Comment:
I tried wrapping `ErrorTopBannerFactory`'s view in a `UIViewRepresentable` but the view could not resolve the constraints properly when added to the dashboard view. I decided to rewrite the banner in SwiftUI instead as it's a fairly simple view.